packages/framework-core/src/services/read-model-store.ts

```diff
@@ -132,6 +134,8 @@ export class ReadModelStore {
       ...migratedReadModel?.boosterMetadata,
       version: currentReadModelVersion + 1,
       schemaVersion: schemaVersion,
+      lastUpdateAt: new Date().toISOString(),
+      lastProjectedEntity: entityTypeName,
```

Comment:
I've been thinking about this, and I think it would be ideal to leverage the fact that you are modifying this to add more information about the entity that can be super useful to debug problems. My suggestion is to change `lastProjectedEntity` by `lastProjectionInfo` and change it to an object with the following params
```typescript
lastProjectionInfo: {
  entityName: "..." // The name of the entity
  entityID: "..." // The ID of the entity
  entityUpdatedAt: entitySnapshotEnvelope.createdAt // Super useful, so we can know the value of the entity used in the last projection.
  projectionMethod: `${projectionMetadata.class.name}.${projectionMetadata.methodName}`
}
```

This way, we can know exactly which projection was last executed, for which entity, and its value (through the "entityUpdatedAt" parameter).